<ui version="4.0" >
 <class>WdgUnsharp</class>
 <widget class="QWidget" name="WdgUnsharp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>81</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="1" >
    <widget class="QSpinBox" native="1" name="intHalfSize" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Half-size:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" native="1" name="doubleAmount" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Threshold:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>Amount:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" native="1" name="intThreshold" />
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
 </customwidgets>
 <includes>
 </includes>
 <resources/>
 <connections/>
</ui>
